Code/Source/Lidar/LidarRaycaster.cpp

```diff
@@ -139,26 +134,25 @@ namespace RGL
 
         if (pointsExpected)
         {
-            m_raycastResults.m_points.resize(m_rglRaycastResults.GetCount());
+            m_raycastResults.m_points.resize(m_rglRaycastResults.m_xyz.size());
         }
 
         if (distanceExpected)
         {
-            m_raycastResults.m_ranges.resize(m_rglRaycastResults.GetCount());
+            m_raycastResults.m_ranges.resize(m_rglRaycastResults.m_distance.size());
         }
 
         size_t usedPointIndex = 0LU;
+        const size_t ResultsSize = pointsExpected ? m_raycastResults.m_points.size() : m_raycastResults.m_ranges.size();
```

Comment:
For constants, such notation (PascalCase) is reserved for global constants - this is what I have observed in O3DE. Local constants simply go according to camelCase.
I am not sure about static const class members - if you find some some example in O3DE, please simply stick to the same convention.

Reply:
The local constants should use PascalCase as described in the [o3de code style guidelines](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#constant-names).